supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -78,6 +78,19 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[discrete]
+[[basic-http-authentication]]
+==== image:images/yes.png[yes] Basic HTTP authentication (noun)
+*Description*: _Basic HTTP authentication_, or _Basic authentication_, is an authorization scheme that uses Hypertext Transfer Protocol (HTTP) to send login credentials as user ID/password pairs that are encoded in Base64. 
```

Comment:
Not critical, but I don't think you need to spell out HTTP. It's listed as one of the easy known abbreviations (and we don't spell it out anywhere else in the SSG).

Reply:
Agree with @bergerhoffer

Reply:
Thanks! Fixed.